src/cloud/nabu-casa-account/log-out-all-devices.md

```diff
@@ -8,11 +8,14 @@ zendesk:
 
 You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, and browsers.
 
->**Notice**: **Loss of remote connection**.
->
-> - If you log out of your device, your remote connection stops working
->
-> - You need physical access to Home Assistant to re-establish the remote connection.
+{% hazard "notice" %}
+
+**Loss of remote connection if you don't have physical access**
+
+- If you log out of your device, your remote connection stops working
+- You need physical access to Home Assistant to re-establish the remote connection.
```

Comment:
This is not only for Home Assistant but also for your other places like the assistants (Alexa/Google) and browsers (like kiosks).
Yes, not related, but worth mentioning.